eternalcore-core/src/main/java/com/eternalcode/core/feature/home/homeadmin/PlayerHomeEntry.java

```diff
@@ -0,0 +1,7 @@
+package com.eternalcode.core.feature.home.homeadmin;
+
+import com.eternalcode.core.feature.home.Home;
+import org.bukkit.entity.Player;
+
+record PlayerHomeEntry(Player player, Home home) {
```

Comment:
that can be memory leak from dependencies in Player structure? Maybe im wrong

Reply:
I don't think so